docs/drash/v2.x/2_tutorials/6_services/5_drash_approved_services/rate_limiter.md

```diff
@@ -0,0 +1,183 @@
+# Rate Limiter
+
+This service will add rate limits to requests from an IP address.
+
+## Table of Contents
+
+- [Before You Get Started](#before-you-get-started)
+- [Folder Structure End State](#folder-structure-end-state)
+- [Steps](#configuration)
```

Comment:
Crap this needs updating too, i based these new service docs off of an existing file so I reckon there’s another file that does this eg paladin or csrf